units/src/amount/error.rs

```diff
@@ -157,18 +158,18 @@ impl OutOfRangeError {
     /// Returns the minimum and maximum allowed values for the type that was parsed.
     ///
     /// This can be used to give a hint to the user which values are allowed.
-    pub fn valid_range(&self) -> (i64, u64) {
+    pub fn valid_range(self) -> (i64, u64) {
```

Comment:
Its late here but these changes are, if I'm not confused, new and nothing like them was ever part of this PR (I think). They showed up just now as clippy warnings in CI.  Two more instances below.